cache/cache.go

```diff
@@ -95,14 +91,8 @@ func (fs FSCache) GetLayer(layerID string) (types.LayerInfo, error) {
 	return layerInfo, nil
 }
 
-func (fs FSCache) getLayer(decompressedBucket, layerBucket *bolt.Bucket, layerID string) (types.LayerInfo, error) {
-	// get a decompressed layer ID
-	d := decompressedBucket.Get([]byte(layerID))
-	if d != nil {
-		layerID = string(d)
-	}
-
-	b := layerBucket.Get([]byte(layerID))
+func (fs FSCache) getLayer(layerBucket *bolt.Bucket, diffID string) (types.LayerInfo, error) {
+	b := layerBucket.Get([]byte(diffID))
```

Comment:
Can `b` be `nil`?

Reply:
It makes sure buckets have been created here.
https://github.com/aquasecurity/fanal/blob/26afaa6f737011f7a821e7df4007cda69b5daaa4/cache/cache.go#L60-L64

If it fails to create these buckets, `NewFsCache` is supposed to fail before calling `getLayer`.